packages/lazy-file/src/fs.test.ts

```diff
@@ -218,3 +218,47 @@ describe('findFile', () => {
     teardown()
   })
 })
+
+describe('writeFile', () => {
+  let tmpDir: string
+
+  function setup() {
```

Comment:
Let's use `beforeEach` and `afterEach` instead of calling `setup()` and `teardown()` manually in each test case below